databases/week3/queries.sql

```diff
@@ -0,0 +1,243 @@
+-- Get all meals
+SELECT
+	title
+FROM
+	meal;
+-- Add a new meal
+INSERT
+	INTO
+	meal(title,
+	description,
+	location,
+	`when`,
+	max_reservations,
+	price)
+VALUES ('Pasta',
+'awful pasta',
+'Copenhagen',
+'2023-10-10 20:00:00',
+10,
+90.99);
+-- Get a meal with any id, fx 1
+SELECT
+	title
+FROM
+	meal
+WHERE
+	id = 1;
+-- Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE
+	meal
+SET
+	description = 'delicious pasta'
+WHERE
+	id = 1;
+-- Delete a meal with any id, fx 1
+DELETE
+FROM
+	meal
+WHERE
+	id = 1;
+-- Get all reservations
+SELECT
+	*
+FROM
+	reservation;
+-- Add a new reservation
+INSERT
+	INTO
+	reservation(number_of_guests,
+	meal_id,
+	contact_phonenumber,
+	contact_name,
+	contact_email)
+VALUES (5,
+1,
+'10203040',
+'John Smith',
+'smith@email.com');
+-- Get a reservation with any id, fx 1
+SELECT
+	*
+FROM
+	reservation
+WHERE
+	id = 1;
+-- Update a reservation with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE
+	reservation
+SET
+	number_of_guests = 3
+WHERE
+	id = 1;
+-- Delete a reservation with any id, fx 1
+DELETE
+FROM
+	reservation
+WHERE
+	id = 1;
+-- Get all reviews
+SELECT
+	*
+FROM
+	review;
+-- Add a new review
+INSERT
+	INTO
+	review(title,
+	description,
+	meal_id,
+	stars)
+VALUES('amazing',
+'an amazing meal',
+1,
+5);
+-- Get a review with any id, fx 1
+SELECT
+	*
+FROM
+	review
+WHERE
+	id = 1;
+-- Update a review with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE
+	review
+SET
+	title = 'excellent'
+WHERE
+	id = 1;
+-- Delete a review with any id, fx 1
+DELETE
+FROM
+	review
+WHERE
+	id = 1;
+-- Add a couple of different meals, reservations and reviews with different attributes
+INSERT
+	INTO
+	meal(title,
+	description,
+	location,
+	`when`,
+	max_reservations,
+	price)
+VALUES ('Rød grød med fløde',
+'sweet dessert',
+'Roskilde',
+'2023-06-18 18:00:00',
+2,
+30.99),
+('Salad',
+'salad with seasonal vegetables',
+'Ringsted',
+'2023-12-12 19:30:00',
+20,
+60.00)
+;
+
+INSERT
+	INTO
+	reservation(number_of_guests,
+	meal_id,
+	contact_phonenumber,
+	contact_name,
+	contact_email)
+VALUES (1,
+2,
+'10203040',
+'John Smith',
+'smith@email.com'),
+(7,
+3,
+'12345678',
+'Jane Doe',
+'jane@email.com')
+;
+
+INSERT
+	INTO
+	review(title,
+	description,
+	meal_id,
+	stars)
+VALUES('fine',
+'the meal was mediocre',
+2,
+2),
+('bad',
+'an awful meal',
+3,
+1);
+-- Get meals that has a price smaller than a specific price fx 90
+SELECT
+	title
+FROM
+	meal
+WHERE
+	price <= 90;
+-- Get meals that still has available reservations
+SELECT
+	title
+FROM
+	meal
+WHERE
+	max_reservations > (
+	SELECT
+		SUM(number_of_guests)
+	FROM
+		reservation
+	WHERE
+		reservation.meal_id = meal.id);
```

Comment:
Technically nothing is wrong, but I like to see an aggregated query showing the key columns instead of only the meal title. 
Try something like this 
```
SELECT meal.title,meal.max_reservations, sum(reservation.number_of_guests) AS Total_reserved_yet
FROM meal
JOIN reservation ON meal.id = reservation.meal_id
GROUP BY reservation.meal_id
HAVING meal.max_reservations > Total_reserved_yet;
```